src/completed-list/completed-list.component.tsx

```diff
@@ -34,11 +34,88 @@ import {
 import styles from "./completed-list.scss";
 import { getStatusColor } from "../utils/functions";
 
-interface CompletedlistProps {
+interface CompletedListProps {
   fulfillerStatus: string;
 }
 
-const CompletedList: React.FC<CompletedlistProps> = ({ fulfillerStatus }) => {
+interface TableRowProps {
+  entry: {
+    uuid: string;
+    orderNumber: string;
+    accessionNumber: string;
+    concept: { display: string };
+    action: string;
+    fulfillerStatus: string;
+    orderer: { display: string };
+    urgency: string;
+    dateActivated: string;
+    patient: { display: string };
+  };
+}
+
+const StatusTag: React.FC<{ fulfillerStatus: string }> = ({
+  fulfillerStatus,
+}) => {
+  return (
+    <Tag>
+      <span
+        className={styles.statusContainer}
+        style={{ color: `${getStatusColor(fulfillerStatus)}` }}
+      >
+        <span>{fulfillerStatus}</span>
+      </span>
+    </Tag>
+  );
+};
+
+const CustomTableRow: React.FC<TableRowProps> = ({ entry }) => {
+  const {
+    uuid,
+    orderNumber,
+    accessionNumber,
+    concept,
+    action,
+    fulfillerStatus,
+    orderer,
+    urgency,
+    dateActivated,
+    patient,
+  } = entry;
+
+  return (
+    <TableRow key={uuid}>
+      <TableCell>
+        <span>{formatDate(parseDate(dateActivated))}</span>
+      </TableCell>
```

Comment:
add class to make the cell display date on a single line

Reply:
@sherrif10 and @pirupius hopefully the actions button will also be in a single line